client/js/components/map/admin/AdminLayerList.vue

```diff
@@ -137,6 +137,7 @@
         <h3 class="mt-4">
           {{ Translator.trans('map.bases') }}
         </h3>
+        <h5>{{ Translator.trans('info.gislayer.delete') }}</h5>
```

Comment:
Here is a permission-Check missing.
```suggestion
        <h5 v-if="hasPermission('feature_map_required')">
          {{ Translator.trans('info.gislayer.delete') }}
        </h5>
```

Reply:
oh yee thx